src/pages/es/guides/deploy/deno.md

```diff
@@ -158,7 +158,7 @@ Si tu proyecto está almacenado en GitHub, la [página web de Deno Deploy](https
         "start": "astro dev",
         "build": "astro build",
         "preview": "deno run --allow-net --allow-read --allow-env ./dist/server/entry.mjs"
```

Comment:
It's missing a comma here, same in English version, should it be added?

```suggestion
        "preview": "deno run --allow-net --allow-read --allow-env ./dist/server/entry.mjs",
```

Reply:
Yup @fredoist is right, looks like there's an invalid JSON here, in all locales. Maybe it can be tracked as a separate issue?

Reply:
@breadadams and @fredoist thank you for bringing this up. Yes, like @breadadams suggested I think the best way to go about this is to open a separate issue for invalid JSON and correct the original ("en") docs. This way all locales can correct this too. We can merge this PR after this issue is resolved.